Source/MediaInfo/Video/File_Hevc.cpp

```diff
@@ -3245,7 +3245,7 @@ void File_Hevc::VPS_SPS_PPS()
             Element_Begin1("nalUnit");
             int16u nalUnitLength;
             Get_B2 (nalUnitLength,                              "nalUnitLength");
-            if (nalUnitLength<2 || Element_Offset+nalUnitLength>Element_Size)
+            if (nalUnitLength<2 || Element_Offset+nalUnitLength>Element_Size || Element_Offset+nalUnitLength>BS->Remain())

```

Comment:
`BS->Remain()` is in bits and actually not relevant here, it leads to breaks in the analysis of some HEVC streams. I remove this commit from the PR.